Model/Ui/CcConfigProvider.php

```diff
@@ -37,10 +47,40 @@ public function getConfig()
                     'years'  => [OmiseCcConfig::CODE => $this->magentoCcConfig->getCcYears()],
                 ],
                 OmiseCcConfig::CODE => [
-                    'publicKey'      => $this->omiseCcConfig->getPublicKey(),
-                    'offsitePayment' => $this->omiseCcConfig->is3DSecureEnabled()
+                    'publicKey'          => $this->omiseCcConfig->getPublicKey(),
+                    'offsitePayment'     => $this->omiseCcConfig->is3DSecureEnabled(),
+                    'isCustomerLoggedIn' => $this->customer->isLoggedIn(),
+                    'cards'              => $this->getCards(),
                 ],
             ]
         ];
     }
+
+    /**
+     * @return  array
+     */
+    public function getCards()
+    {
+        if (! $this->customer->getMagentoCustomerId() || ! $this->customer->getId()) {
+            return array();
```

Comment:
Still inconsistent use of short and long array syntax around

Reply:
@jonrandy there are short syntax in `getConfig` that doesn't relate with the pull request's objective, I'll have another quick pull request to update all of short syntax ones to the long array syntax. 
Is that sounds ok?